index.js

```diff
@@ -112,6 +112,24 @@ class Datastore {
         return Promise.reject(new Error('Not implemented'));
     }
 
+    /**
+     * Run raw query on the datastore
+     * @method query
+     * @param  {Object}        config                Configuration object
+     * @param  {Array<Object>} [config.queries]      Map of database type to query
+     * @param  {String}        [config.table]        Table name
+     * @param  {Object}        [config.replacements] Parameters to replace in the query
+     * @param  {Boolean}       [config.rawResponse]  Return raw response without binding to model
+     */
+    query(config) {
+        return validate(config, datastoreSchema.query)
+            .then(validConfig => this._query(validConfig));
+    }
+
+    _query() {
+        return Promise.reject(new Error('Not implemented'));
```

Comment:
If we're going to throw error if not defined, should probably bump major version

Reply:
Updated PR description to reflect this^